.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,60 @@
+name: Reusable action of Integration tests
+
+on: 
+  workflow_call:
+    secrets:
+      CI_USER_TOKEN:
+        required: true
+      TRAVIS_COM_TOKEN:
+        required: true
+      PREP:
+        required: false
+      RELEASE:
+        required: false
+
+jobs:
+  integration_tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        # You should create a personal access token and store it in your repository
+        token: ${{ secrets.CI_USER_TOKEN }}
+        repository: 'optimizely/travisci-tools'
+        path: 'home/runner/travisci-tools'
+        ref: 'master'
+    - name: set SDK Branch if PR
+      if: ${{ github.event_name == 'pull_request' }}
+      run: |
+        echo "SDK_BRANCH=${{ github.head_ref }}" >> $GITHUB_ENV
```

Comment:
what does this line do?

Reply:
In case of pull request, it exports the PR branch name to github env variables as `SDK_BRANCH`. This is required by `travisci-tools/trigger-script-with-status-update.sh`